firstvoices/backend/serializers/site_serializers.py

```diff
@@ -42,6 +44,22 @@ class Meta(LinkedSiteSerializer.Meta):
         fields = LinkedSiteSerializer.Meta.fields + ("logo", "features")
 
 
+@extend_schema_serializer(
```

Comment:
would this exclude these fields from the docs page ? can we not specify types for each fields as well ?
like list of category model in categories

Reply:
Yes, it does exclude these fields from the docs page. Currently, they are not being generated anyway and a warning was being thrown. I agree that we should go through the docs and fill them in completely, but I think this should be a low priority right now. One option would be to close this PR and leave the mass of warnings as a reminder to update the docs later.

Reply:
I think it's great to get rid of the warnings for now. @NolanVH could you please log a ticket in the backlog to deal with these missing fields in the future?